lib/static/css/main.css

```diff
@@ -935,14 +942,20 @@ input#search_input_react {
 input#search_input_react:focus,
 input#search_input_react:active {
   color: #fff;
-  width: 220px;
 }
 
 @media only screen and (max-width: 735px) {
   .navSearchWrapper {
     width: 40%;
   }
 }
+@media only screen and (max-width: 1010px) {
```

Comment:
Suggestion to follow a mobile-first approach:

```
/* first define the base styles */
.navSearchWrapper {
  display: none;
}
/* override when certain breakpoint is reached (1010px in this case) */
@media only screen and (min-width: 1010px) {
  .navSearchWrapper {
    display: none;
  }
 }
```
I see the codebase uses `max-width` `media-queries` so maybe mixing up is not the best thing, your call, I did want to suggest here because we follow a _mobile-first_ convention in the other repos.

Reply:
Yes, I think is best to follow a mobile-first approach, but I would do it in a separate refactor PR, modifying all media queries at once